src/pymmcore_plus/core/_sequencing.py

```diff
@@ -228,6 +228,12 @@ def can_sequence_events(
     def _nope(reason: str) -> tuple[bool, str] | bool:
         return (False, reason) if return_reason else False
 
+    # Action
+    if (e1.action is None or not isinstance(e1.action, AcquireImage)) or (
+        e2.action is None or not isinstance(e2.action, AcquireImage)
+    ):
```

Comment:
```suggestion
    # Action
    if not isinstance(e1.action, (AcquireImage, type(None))) or not isinstance(
        e2.action, (AcquireImage, type(None))
    ):
```

this doesn't feel quite right.  I thought the goal was to *allow* `None`, not to forbid it.